src/kernel/mconfig.ml

```diff
@@ -1021,7 +1021,9 @@ let cmt_path config =
   let stdlib = stdlib config in
   let exp_dirs = List.map ~f:(Misc.expand_directory stdlib) dirs in
   let stdlib = if config.ocaml.no_std_include then [] else [ stdlib ] in
-  config.query.directory :: List.rev_append exp_dirs stdlib
+  let result' = List.rev_append exp_dirs stdlib in
```

Comment:
Nit: name it `result` and not `result'`. I'd say this even if there was a `result` in scope, but there does not appear to be.

Reply:
Done